test/extended/openstack/cpms.go

```diff
@@ -139,3 +169,14 @@ func getControlPlaneMachineSet(ctx context.Context, dc dynamic.Interface) (objx.
 	}
 	return cpmsList[0], nil
 }
+
+type FailureDomain struct {
+	ComputeZone string
+	VolumeZone  string
+	VolumeType  string
+}
+
+// Duplicate a FD
+func duplicateFd(fdList *[]FailureDomain) {
```

Comment:
Please define this function locally to the test function

```Go
duplicateFD := func(fdList *[]FailureDomain) {
```

Reply:
I'm replacing it with the line of code instead